R/pkg/R/functions.R

```diff
@@ -54,7 +54,8 @@ setMethod("lit", signature("ANY"),
 #' @name abs
 #' @family normal_funcs
 #' @export
-#' @examples \dontrun{abs(df$c)}
+#' @examples
+#' \dontrun{abs(df$c)}
```

Comment:
I'm actually not sure about this change - if the example is simple and is single line, I think they can go on the same line as `@example` tag.
More importantly though, I don't think majority of SQL function examples like this is usable or useful actually, and would rather have multi-line "real" example